topologies/datacenter-latest/labguides/source/cvp-vxlan.rst

```diff
@@ -98,63 +123,63 @@ VxLAN
     :align: center
     :width: 50% 
 
-* Click **save** on the Network Provisioning main view
-    .. note: a **Task** will be generated
+* Click **Save** on the **Network Provisioning** main view
+   **Note:** a **Task** will be generated
 
 .. image:: images/cvp-vxlan/leaf3-vxlan-configlet-main-save.png
     :align: center
     :width: 50% 
 
 5. Create a **Change Control** with the generated Task
 
-* click **Tasks** from the left selection column
+* Click **Tasks** from the left selection column
 
-* click the checkbox next to the generated task
+* Click the checkbox next to the generated task from the pool of **Assignable Tasks**
```

Comment:
```suggestion
* Click the checkbox next to the generated task from the pool of Assignable Tasks
```

Reply:
Where does the notion, come from that bolding is only for clickable items? I am going to attempt to create a ATD labguide style guide to formalize the conventions we are using.